Services (#243)
northeastern-rc/rc-public-documentation: docs/source/welcome/services/index.md

(servicesprovided)=
# Services we Provide

::::{grid} 1 2 2 3
:gutter: 1 1 1 2

:::{grid-item-card} {octicon}`device-desktop;1.5em;screen-full` Compute
:link: ../../hardware/hardware_overview
:link-type: doc

> An overview of our computational resources.

Reviewer: ```suggestion
An overview of available computational resources.
```
If "available" doesn't make sense, you can just remove that, or change to something else. (Per ITS Comms, we aren't supposed to use we/our)

+++
[Learn more »](../../hardware/hardware_overview)
:::

:::{grid-item-card} {octicon}`database;1.5em;sd-mr-1` Storage
:link: ../../datamanagement/discovery_storage
:link-type: doc

> We offer several storage solutions for active and archival use.

Reviewer: ```suggestion
RC offers several storage solutions for active and archival use.
```

+++
[Learn more »](../../datamanagement/discovery_storage)
:::

> :::{grid-item-card} {octicon}`mortar-board;1.5em;sd-mr-1` Classroom

Reviewer: ```suggestion
:::{grid-item-card} {octicon}`mortar-board;1.5em;sd-mr-1` Courses
```

:link: ../../classroom/class_use
:link-type: doc

> We support classroom use of HPC resources.

Reviewer: ```suggestion
Thinking about using HPC resources in your course, review the courses FAQ.
```
We should probably try to stick to using 'course' rather than 'classroom' for consistency and clarity.

+++
[Learn more »](../../classroom/class_use)
:::
:::{grid-item-card} {octicon}`device-camera-video;1.5em;sd-mr-1` Consultations
:link: ../services/consultations
:link-type: doc

One-on-one support as you access the cluster.
+++
[Learn more »](../services/consultations)
:::
:::{grid-item-card} {octicon}`play;1.5em;sd-mr-1` Training
:link: ../services/training
:link-type: doc

> A wide variety of free training sessions.

Reviewer: ```suggestion
RC provides a wide variety of free training sessions.
```

+++
[Learn more »](../services/training)
:::
:::{grid-item-card} {octicon}`person-add;1.5em;sd-mr-1` Collaborations
:link: ../services/collaboration
:link-type: doc

> We look forward to collaborations with researchers.

Reviewer: ```suggestion
Collaborate with RC on your research.
```

+++
[Learn more »](../services/collaboration)
:::
::::

> If you feel your research has benefited from support from RC @ NU or want to acknowledge the cluster please use one of the statements below:

Reviewer: ```suggestion
If your research has benefited from support from the RC team, or you want to acknowledge the cluster, please use one of the statements below:
```
